tensorflow/c/experimental/filesystem/modular_filesystem.cc

```diff
@@ -224,9 +224,10 @@ Status ModularFileSystem::DeleteFile(
   return StatusFromTF_Status(plugin_status.get());
 }
 
-Status ModularFileSystem::DeleteRecursively(
-    const std::string& dirname, int64* undeleted_files,
-    int64* undeleted_dirs /*, TransactionToken* token */) {
+Status ModularFileSystem::DeleteRecursively(const std::string& dirname,
+                                            int64* undeleted_files,
+                                            int64* undeleted_dirs,
+                                            TransactionToken* token) {
```

Comment:
For these methods `token` is an input argument, right? It should come before `undeleted_*` which are outputs